install-dependencies.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: localhost
```

Comment:
This should be `hosts: all`, matching the test nodes. `localhost` is the executor where tasks like installing pkgs are not allowed.